modules/cart/src/Plugin/Block/CartBlock.php

```diff
@@ -74,7 +75,7 @@ public static function create(ContainerInterface $container, array $configuratio
    */
   public function defaultConfiguration() {
     return [
-      'item_text' => $this->t('items'),
+      'item_text' => 'item' . PluralTranslatableMarkup::DELIMITER . 'items',
```

Comment:
Went to, and got code violation warnings for concatenating inside of a translate function. There's no core pattern here [that I can see] to follow.